src/index.js

```diff
@@ -0,0 +1,34 @@
+import './style.css';
+import fetchData from './modules/getData.js';
+import postData from './modules/postData.js';
+
+const refresh = document.querySelector('.refresh-btn');
+const form = document.getElementById('form');
+const userInput = document.getElementById('user');
+const scoreInput = document.getElementById('score');
+const text = document.querySelector('.text');
+
+// submit form
+form.addEventListener('submit', (e) => {
+  e.preventDefault();
+  postData();
+  if (userInput.value === '' || scoreInput.value === '') {
+    alert('Please Enter a valid data');
+  }
```

Comment:
- [ ] [OPTIONAL] Please consider not using `alert`. It is regarded as bad design because it blocks the entire browser and code execution. Also, the look and feel will not be consistent as JavaScript uses native browser console for `alert`.
Present the message unobtrusively inline for informative messages that do not require action as in this case 😎.